extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/FastCamelContext.java

```diff
@@ -81,7 +81,10 @@ public String getVersion() {
     @Override
     protected Registry createRegistry() {
         // Registry creation is done at build time
-        throw new UnsupportedOperationException();
+        throw new UnsupportedOperationException(
+                "In case that the test based on CamelQuarkusTestSupport throws this exception, " +
+                        "be aware that re-starting of context is not possible. " +
+                        "(See https://camel.apache.org/camel-quarkus/2.10.x/user-guide/testing.html)");
```

Comment:
I add a text into the `UnsupportedOperationException`, which is thrown in case that context is re-started with `CamelQuarkusTestSupport`. This will make the exception understandable for users. @ppalaga What do you think?

Reply:
We need to be a bit careful about linking to docs because 2.10.x is not LTS and will get removed at some point.

Personally, I would just throw a plain `UnsupportedOperationException` without the message.

We could add some notes into the docs about the registry. E.g if you need to bind beans to it, then you can: 

* Use CDI producers
* Use the Camel `@BindToRegistry` annotation
* `@Inject` the `CamelContext` into the test and get a handle on the registry from it

Reply:
I removed the link and kept the rest, as we discussed,